backends/platform/android/org/scummvm/scummvm/ShortcutCreatorActivity.java

```diff
@@ -55,6 +55,34 @@ public class ShortcutCreatorActivity extends Activity {
 
 	private IconsCache _cache;
 
+	static void pushShortcut(Context context, String gameId, Intent intent) {
+		Map<String, Map<String, String>> parsedIniMap;
+		try (FileReader reader = new FileReader(new File(context.getFilesDir(), "scummvm.ini"))) {
+			parsedIniMap = INIParser.parse(reader);
+		} catch(FileNotFoundException ignored) {
+			parsedIniMap = null;
+		} catch(IOException ignored) {
+			parsedIniMap = null;
+		}
+
+		if (parsedIniMap == null) {
+			return;
+		}
+
+		Game game = Game.loadGame(parsedIniMap, gameId);
```

Comment:
We should check for game being returned as "null" here.

Reply:
Fixed, thank you!